tests/testthat/test-nmxml.R

```diff
@@ -279,6 +279,35 @@ test_that("NONMEM estimates from nmext", {
                "'arg' should be one of ")
 })
 
+test_that("NONMEM estimates from nmext - multiple tables", {
+  project <- system.file("nonmem", package="mrgsolve")
+  
+  a <- mrgsolve:::nmext(run = 2005, project = project, index = "last")
+  aa <- attributes(read_nmext(run = 2005, project = project, index = "last"))
+  expect_equal(aa$index,5)
+  expect_match(aa$table, "First Order Conditional")
+  
+  b <- mrgsolve:::nmext(run = 2005, project = project, index = 2)
+  ab <- attributes(read_nmext(run = 2005, project = project, index = 2))
+  expect_equal(ab$index,2)
+  expect_match(ab$table, "Importance Sampling")
+  
+  expect_true(a$theta$THETA3 != b$theta$THETA1)
```

Comment:
Did you mean to compare different thetas from different tables here?

Reply:
Thanks; no, that's a mistake.